modules/installation-localzone-generate-k8s-manifest.adoc

```diff
@@ -26,8 +26,53 @@ $ ./openshift-install create manifests --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the installation directory that
 contains the `install-config.yaml` file you created.
 
-. Create the machine set manifests for the worker nodes in your Local Zone.
+. Set the default Maximum Transmission Unit (MTU) according to the network plugin:
++
+[IMPORTANT]
+====
+Generally, the Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300. See link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation. 
+The cluster network MTU must be always less than the EC2 MTU to account for the overhead. The specific overhead is determined by your network plugin, for example:
+
+- OVN-Kubernetes: `100 bytes`
+- OpenShift SDN: `50 bytes`
+
+The network plugin could provide additional features, like IPsec, that also must be decreased the MTU. Check the documentation for additional information.
+
+====
+
+.. If you are using the `OVN-Kubernetes` network plugin, enter the following command:
++
+[source,terminal]
+----
+$ cat <<EOF > <installation_directory>/manifests/cluster-network-03-config.yml
```

Comment:
@patrickdillon @yunjiang29 I just found that the installer is creating the file [`cluster-network-03-config.yml`](https://github.com/openshift/installer/blob/master/pkg/asset/manifests/network.go#L21), the good news is it is only for P/Z platform. Do you think that we need to keep adding this `03` file or we must to change to `99`?

```suggestion
$ cat <<EOF > <installation_directory>/manifests/cluster-network-99-config.yml
```

Reply:
Closing this thread as it does not affect AWS, and phase1 we must have this fixed.